src/Solver/Interoperability.cpp

```diff
@@ -309,7 +309,22 @@ void c_interoperability_report_device_memory_status() {
     }
   }
 
-  double c_interoperability_M2invDiagonal(int no) {
+void c_interoperability_numberOfTriangleQuadraturePoints(int* n) {
+  *n = dr::misc::numberOfBoundaryGaussPoints;
+}
+
+void c_interoperability_triangleQuadratureRule(double* points, double* weights) {
+  constexpr size_t numberOfPoints = dr::misc::numberOfBoundaryGaussPoints;
+  auto pointsView = init::quadpoints::view::create(const_cast<real *>(init::quadpoints::Values));
```

Comment:
out of curiosity, where in the code are defined init::quadpoints::Values?

Reply:
They are created by yateto